pkg/cli/admin/upgrade/status/mco/layered_node_state.go

```diff
@@ -80,7 +81,11 @@ func (l *LayeredNodeState) isImageAnnotationEqualToPool(anno string, mcp *mcfgv1
 	if lps.IsLayered() && lps.HasOSImage() {
 		// If the pool is layered and has an OS image, check that it equals the
 		// node annotations' value.
-		return lps.GetOSImage() == val
+		if lps.GetOSImage() == val {
+			return true
+		}
+		l.ReasonOfUnavailability = fmt.Sprintf("the pool has an OS iamge %s that does not equal the one from annotation %s", lps.GetOSImage(), val)
```

Comment:
nit: "iamge" - "image".

And since we're complaining about the node, maybe:

```go
fmt.Sprintf("the node annotation %s is %s, while the pool is trying to reconcile OS image %s", anno, lps.GetOSImage(), val)
```

or something?  Because "the one from the annotation" isn't giving the user many breadcrumbs if it turns out they want to investigate further.

Reply:
Done